src/App.js

```diff
@@ -24,15 +26,32 @@ class App extends Component {
     this.setState({ movies: movieData.movies });
   }
 
+  toggleHighlighted = (e) => {
+    if (this.state.selectedMovie) {
+      this.setState({selectedMovie: false})
+    } else {
+      let highlightedMovie = this.state.movies.find(movie => movie.id == e.target.id);
+      this.setState({ selectedMovie: highlightedMovie })
+    }
+  }
+
   render() {
     return (
-      <div className='app'>
+      <div className='app' >
         <Header />
-        {this.state.movies.length &&
+        {this.state.movies.length && !this.state.selectedMovie &&
```

Comment:
I like the relative simplicity of the `&&` use here. I think if you had even more logic that required another `&&`, then I think it would quickly get harder to determine at a quick glance hat will show on the page. If that were the case, how would you consider refactoring? You don't have to answer here, but it's something to think about.